src/main/features/eligibility/model/claimType.ts

```diff
@@ -2,7 +2,8 @@ export class ClaimType {
 
   static readonly PERSONAL_CLAIM = new ClaimType('PERSONAL_CLAIM', 'Just myself or my organisation')
   static readonly MULTIPLE_CLAIM = new ClaimType('MULTIPLE_CLAIM', 'More than one person or organisation')
-  static readonly REPRESENTATIVE_CLAIM = new ClaimType('REPRESENTATIVE_CLAIM', 'A client - I’m their legal representative')
+  // We are not using " I’m " as the apostrophe causes errors with the cookie
```

Comment:
Maybe there is some escaping missing? Have you tried the html entity (`&apos;`) as well?

Reply:
The problem was that `’` was escaped into `&apos;` and last char (`;`) was considered as cookie delimiter which in the end made json invalid.

As a solution, by convention, I exclude `displayValue` from serialisation process.